maven-embedder/src/main/java/org/apache/maven/cli/MavenCli.java

```diff
@@ -19,6 +19,7 @@
  * under the License.
  */
 
+import com.google.common.annotations.VisibleForTesting;
```

Comment:
This looks like a guava class. We want to avoid guava in our codebase, and only keep it as a transitive dependency for sisu(?). A comment explaining its visibility is good enough

Reply:
Replaced the annotation with a comment.